app/helpers/admin/site_maps_helper.rb

```diff
@@ -3,12 +3,26 @@
 module Admin::SiteMapsHelper
   include Admin::AdminHelper
 
+  def sitemap_pages
+    @sitemap_pages ||= Page.where(:_type.in => %w(Page SetPage)).entries
```

Comment:
Why `where(:_type.in => %w(Page SetPage)`? Will this break in apps where `Page` has been subclassed?